Doc/library/subprocess.rst

```diff
@@ -689,7 +689,10 @@ execute, will be re-raised in the parent.
 
 The most common exception raised is :exc:`OSError`.  This occurs, for example,
 when trying to execute a non-existent file.  Applications should prepare for
-:exc:`OSError` exceptions.
+:exc:`OSError` exceptions. Note that, when ``"shell=True"``, :exc:`OSError` will
```

Comment:
please wrap the "will" so that the line does not exceed 80 chars.

Reply:
Ok, I have done that. In a separate additional commit I also went over the whole file line wrapping any other paragraphs that went over.

Reply:
Please remove the separate commit - it spams the diff and makes it hard to see what you changed.

In general we don't do sweeping fixes like that, but try not to add more formatting problems with the changes we commit.

Reply:
Ok, it's been reverted.